### PR TITLE
Add bulleted list of SuperAdmin priveleges

### PR DESCRIPTION
**Pull request by ksmith94**
This will add a list of SuperAdmin privileges to the Projects doc. Related to issue #3407

### CODE REVIEW

File: packages/docs/docs/access/projects.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ While they _do_ cross the [`Project`](/docs/api/fhir/medplum/project) isolation
 
 The main exception to this isolation model is the "Super Admin" project. This is a special project that provides a global view over all the resources on the Medplum server. See our [SuperAdmin Guide](/docs/self-hosting/super-admin-guide) for more information.
 
+The SuperAdmin has the following privileges:
+
+- Access to protected resources.
+- Ability to overwrite the `id` of a resource, which is normally server generated.
+- Ability to overwrite fields in the `meta` element of resources such as `author`, `lastUpdated`, etc.
+- Bypasses schema type validation (e.g. write numbers to a string field). This can be turned off by enabling strict mode in your SuperAdmin project.
```
Comment (mattwiller) [inline packages/docs/docs/access/projects.md:41]: This is no longer the default  after v2.1.18

File: packages/docs/docs/access/projects.md [hunk 2]
```diff
@@ -33,6 +33,13 @@ While they _do_ cross the [`Project`](/docs/api/fhir/medplum/project) isolation
 
 The main exception to this isolation model is the "Super Admin" project. This is a special project that provides a global view over all the resources on the Medplum server. See our [SuperAdmin Guide](/docs/self-hosting/super-admin-guide) for more information.
 
+The SuperAdmin has the following privileges:
```
Comment (mattwiller) [inline packages/docs/docs/access/projects.md:36]: This should probably be "Super Admin" for consistency with above

File: packages/docs/docs/access/projects.md (hunk 1 above)
Comment (rahul1) [inline packages/docs/docs/access/projects.md:41]:
```suggestion
```

**Review by rahul1 — APPROVED** [review]
Approved **pending the removal of the last bullet**

**Pull request merged**